openapi.yaml

```diff
@@ -3027,6 +3140,40 @@ components:
           type: string
           description: "The phone of the contact"
           example: "402-555-1212"
+    ProjectCollaborator:
+      type: object
+      properties:
+        company_id:
+          type: string
+          description: "The unique ID of the collaborating company"
+          example: "48892885"
+        project_id:
+          type: string
+          description: "The unique ID of the project the collaborator is apart of"
+          example: "48892885"
+        permissions:
+          type: array
+          description: "The permissions for the collaborator"
+          items:
+            type: string
+        sharing_start_at:
```

Comment:
Seems like this should be `accepted_at` since the sharing is historical IIRC

Reply:
I was keeping it closely tied to what the field is on the database, but I agree `accepted_at` makes more sense.

Reply:
@coreycollins `accepted_at` is in the database too